randovania/data/json_data/prime3.txt

```diff
@@ -3881,13 +4745,13 @@ Asset id: 13589229454537797972
 
 > Dock to Defense Access (Balcony); Heals? False
   * Normal Door to Defense Access/Dock to Command Station (Balcony)
-  > Pickup (Missile Expansion)
-      Grapple Voltage
 
 > Pickup (Missile Expansion); Heals? False
   * Pickup 90; Major Location? False
   > Morph Ball Tunnels
       Trivial
+  > Main Room
+      After Command Station Morph Ball Barrier
```

Comment:
Delete Morph Ball Tunnels connection. Change Main Room connection to Defense Access Base node. Add connection to Flux Control as Trivial